scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -302,7 +327,8 @@ private void initFooter() {
                 initSubmitAtPart();
                 rootPage.addMember(messagePanel);
                 rootPage.addMember(submitCancelButtons);
-                startNowRB.setValue(true);
+                startAccordingPlanningRB.setValue(true);
```

Comment:
startAccordingPlanningRB What is the "RB" (Probably RadioGroup) it is not immediately obvious for me. Can you revise the RB in the variable names.

Reply:
@tobwiens "RB" stands for RadioButton. I can have a pass on the variable names and be more explicit (startAtRadioButton). I added the new radio button  "startAccordingPlanningRB" following the naming used for the other radio buttons. 
I will fix this, thanks a lot!

Reply:
I think RB is for Radio Button. startAccordingPlanningRB attribute is the RadioButton type. Moreover on the attributes Zeineb added comments for the description. I don't think it needs more clarification here.